### DISCUSSION TITLE
How to choose the underlying GitHub Copilot model?

### DISCUSSION THREAD

**Discussion by jdcola** [intitni/CopilotForXcode#611] [General]
Seems in Chat it's using GPT, but I want to choose Claude.
Didn't find an option.

**Comment by intitni** [comment]
I don't think there is a proper way to do it through the language server unless we call the endpoint directly. #612

**Comment by intitni** [comment]
A workaround if you wish to change the model right now:

https://github.com/intitni/CopilotForXcode/issues/612#issuecomment-2520765662
